### PR TITLE
change currency unit

### PR DESCRIPTION
**Pull request by nguyenvanman**

### CODE REVIEW

File: ClubManagement/ClubManagement/Activities/MoneyDetailActivity.cs [hunk 1]
```diff
@@ -52,7 +52,7 @@ private void Init()
             moneyId = Intent.GetStringExtra("MoneyId");
             var time = Intent.GetStringExtra("Time");
 
-            tvDescription.Text = $"{time} - Budget : {budget}$\n\n{description}";
+            tvDescription.Text = $"{time} - Budget : {budget * 1000} đ\n\n{description}";
```
Comment (voquanghoa) [inline ClubManagement/ClubManagement/Activities/MoneyDetailActivity.cs:55]: Why we need to `*1000` here?

**Review by voquanghoa — CHANGES_REQUESTED** [review]
Please change the title of this pull request.

File: ClubManagement/ClubManagement/Activities/MoneyDetailActivity.cs (hunk 1 above)
Comment (nguyenvanman) [reply ClubManagement/ClubManagement/Activities/MoneyDetailActivity.cs:55]: currency unit is 1000đ

Comment (voquanghoa) [reply ClubManagement/ClubManagement/Activities/MoneyDetailActivity.cs:55]: No, don't do it. The unit should be `đ`

**Pull request closed without merging**